docs/src/references/child-devices.md

```diff
@@ -0,0 +1,123 @@
+# Child-Device Support
+
+Child-Device support allows to connect devices (e.g. sensors, actors, PLCs, any other kind of device) to the thin-edge device, to represent those devices in the cloud. Furthermore processes running on the thin-edge device can make use of child-device support to appear as a kind of a logical device in the cloud.
+
+## Terminology
+- **thin-edge device**:
+  the physical device thin-edge is running on
+- **external-device**:
+  some kind of physical device that is connected to the _thin-edge device_ (e.g. a PLC, a fieldbus device, a sensor or any other kind of device)
+- **logical child-device**:
+  a process running on the _thin-edge device_ that makes use of thin-edge's child-device API to be represented as individual device in the cloud
+- **child-device**:
+  a _logical child-device_ or an _external-device_ that makes use of thin-edge's child-device API to be represented as individual device in the cloud
+- **child-device twin**:
+  the entity in the cloud, that represents the child-device; in case of C8Y that is a managed object
+- **child-id**:
+  a unique string referencing the _external-device_ or _logical child-device_ to the corresponding _child-device twin_
+- **child-device agent**:
+  that piece of software that makes the glue between the _external-device_ and thin-edge APIs; could be located on the _external-device_ or running as process on the _thin-edge device_
+
+## Principles
+
+Child-device support basically focuses on three aspects:
+   1) provision the child-device, i.e. making thin-edge and the cloud aware of the child-device (external or logical) 
+   2) thin-egde APIs provided on network, to enable external-devices APIs to access them
+   3) associate consumed/provided data of a child-device (external or logical) to it's child-device twin in the cloud
```

Comment:
What I was trying to say is:
* to (1) 

  the child-device must have a chance to say to thin-edge "hello, I am there", and "I am capable for config man, and logging and softward man"
  
* for (2)

  the child-device must be able to access thin-edge/plugins functionality (e.g. cfg man, logging, software man) via network APIs, instead of e.g. thin-edge local filesystem or  local CLI
  
* for (3)

  the child-device must be able tell thin-edge/plugins which child-device twin in the cloud shall be associated with it's functionality/data (e.g. cfg-man, logging, software man, ... measurements, ...) and thin-edge/plugins have to tell that to the cloud.

Reply:
@didier-wenzek ,  whats about that:

Child-device support basically focuses on three aspects:
   1) the child-device can be provisioned to thin-edge, i.e. making thin-edge and the cloud aware of the child-device (external or logical)
   2) the child-device can access all relevant thin-egde APIs on local network (e.g. for telemetry data, config management, software management, ...)
   3) the child-device can specify for each API call a child-device twin in the cloud, to which all data/operations of the call will be associated